### ISSUE TITLE
"Join Group" button, link to Group owner's profile, link to Group Memebers not working

### ISSUE THREAD

**Issue by KevinChagnon** [gctools-outilsgc/gcconnex#643]
user: Dennis.OFarrell

"The "Join group" button is inactive. In fact none of the links inside the white box work. Items above (e.g. Messages) and below (e.g. Blog) work fine.  It does nothing when clicked. Also mouse pointer does not change when hovering over it as with other links

![links not working](https://cloud.githubusercontent.com/assets/23527358/21458572/a3d89226-c905-11e6-9849-9b55ae1fd373.png)

using Internet Explorer 11 (11.0.9600.18524) and FireFox 50.0

Groups where I have seen the behaviour
-	https://gcconnex.gc.ca/groups/profile/22346843/%E2%8D%9Fbig-data-community-communaute-des-megadonnees%E2%8D%9F
-	https://gcconnex.gc.ca/groups/profile/48967/open-source-community
-	https://gcconnex.gc.ca/groups/profile/24345174/data-visualization
-	https://gcconnex.gc.ca/groups/profile/4441183/environment-canada-green-team-equipe-verte-de-environnement-canada
Behaviour even in groups that I am a member of…
-	https://gcconnex.gc.ca/groups/profile/6059737/public-servants-promoting-a-greener-environmentdes-fonctionnaires-pour-un-environnement-plus-vert
-	https://gcconnex.gc.ca/groups/profile/15074965/canadian-environmental-sustainability-indicators-cesi-indicateurs-canadiens-de-durabilite-de-lenvironnement-icde 
Groups that seem to work ok
-	https://gcconnex.gc.ca/groups/profile/272967/blueprint-2020-objectif-2020
-	https://gcconnex.gc.ca/groups/profile/226392/gctools
-	

I’m sure this was not always the case and it does seem to coincide with new versions of browsers "

**Comment by KevinChagnon** [comment]
Additional comments from user:
"I tried to test this again and it seemed to be working ok but then it seemed to revert back to the behaviour. I closed and reopened Internet Explorer and it seemed to work ok again. 

However when I opened FireFox64bit the behaviour resurfaced. Not only that it resurfaced in one of my open IE windows!? I closed and restarted IE and reloaded the url and it worked fine. Same thing happened with FireFox32bit. Which is weird since it seemed to work on Firefox earlier.

All pretty confusing but it seems to be related to multiple browsers used. Or maybe this is all random"

**Comment by Phanoix** [comment]
Those buttons are all just links so they really shouldn't depend on the browser unless it simply stops loading the page part-way on its own. Do other links on those pages work?

**Comment by Phanoix** [comment]
Another question: when this bug shows up, can the user interact with anything inside the group summary area? For example, can the text be selected / highlighted and does the cursor change when you mouse over the text, links, etc.?

**Comment by KevinChagnon** [comment]
I'v e sent these questions to the user. Waiting for a reply.

**Comment by KevinChagnon** [comment]
Q. Do other links on those pages work? (tags or the share button for example) 
A. Yes, all other links/buttons work

Q. When did this start happening? 
A. A bit hard to say as it was intermittent and I didn’t really understand what was going on at first. I think back in November.

Q. When this bug shows up, can the you interact with anything inside the group summary area? 
A. No

Q. Can the text be selected / highlighted and does the cursor change when you mouse over the text, links, etc.? 
A. No (Although I have discovered that if I start selection outside the box, I can select most contents (not the Members button), paste them somewhere and the links work.)

Additional Comments from user:
Also a snippet of where it just happened.  Interestingly, when I clicked on the link (from this email) **[see entry from Dec 23rd]** and when it opened in FireFox (which was not opened yet today) it seemed fine. Once I logged in, however, the group banner was again dormant. I tried closing FireFox and clicking the link again and the problem was still there. 

Opening the url’s in IE seems to work fine (although I have also seen the issue in IE before and there is the problem that action icons like Share on the wire or download are invisible). If I can’t get it to work in FireFox it will be quite annoying since I can save my credentials and navigate to anywhere quickly whereas in IE we are not allowed to save credentials and it is a pain entering passwords all the time (and there are the invisible icons).
